### ISSUE TITLE
entities layout: give less space to secondary info

### ISSUE THREAD

**Issue by maxlath** [inventaire/inventaire-client#457]
such as an author's awards, which sometimes can be many, resulting in hard-to-read/useless information cluttering the top of the page, ex with https://inventaire.io/entity/wd:Q181659:
![Capture d’écran 2023-11-29 à 18 40 03-fullpage rect](https://github.com/inventaire/inventaire-client/assets/1596934/83204417-9b34-4318-b5bf-6234a4e75ec6)

Awards received (`wdt:P166`), influenced by (`wdt:P737`), and probably other similar properties could be turned into things more similar to, or directly reusing, `RelativeEntitiesList`, and be placed lower in the page (somewhere below the sub-entities lists)

**Comment by jum-s** [comment]
These huge text blocks has already been *temporarily* fixed by 8fbd0d14b14aca51903b2094eb455aa8acfd9890 since it also happens to [other infoboxes](https://inventaire.io/entity/wd:Q28667972)

Moving awards received (`wdt:P166`) still feel like a good idea though.

Influenced by (`wdt:P737`) could have its own `relatives-lists`.

**Comment by jum-s** [comment]
Another temporary solution would be to reuse the "show more button" mechanism, since its already in place on infoboxes for editions lists. To test it, one may remove `listDisplay && ` from the following line in `infobox.svelte`:
```
$: wrappedSize = listDisplay && infoboxHeight && infoboxHeight > wrappedInfoboxHeight
```

**Comment by maxlath** [comment]
+ 1 on "influenced by" getting its RelativeEntitiesList

I feel like the website, and other external lists are usually more of interest than the occupation/educated at/awards, so maybe those could be moved up (and made more readable with external sites favicons :D), letting occupation/educated at/awards at the end of that block. Then to wrap that block, maybe a `max-height` + `overflow-y: auto`, like for the summary on the right, rather than a "see more" button?

**Comment by jum-s** [comment]
> rather than a "see more" button?

yes, i am now convinced that "see more" is not a good UX option, i was suggesting it as a temporary quick fix since its already implemented. But a more long term solution would rely on `overflow-y:auto` since its a common pattern.

**Comment by maxlath** [comment]
Addressed by  8fbd0d1
